src/ansys/fluent/core/services/settings.py

```diff
@@ -367,9 +371,13 @@ def get_attrs(self, path: str, attrs: list[str], recursive: bool = False) -> Any
     @_trace
     def has_wildcard(self, name: str) -> bool:
         """Checks whether a name has a wildcard pattern."""
-        return self._scheme_eval.is_defined(
-            "has-fnmatch-wild-card?"
-        ) and self._scheme_eval.scheme_eval(f'(has-fnmatch-wild-card? "{name}")')
+        if FluentVersion(self.scheme_eval.version) < FluentVersion.v252:
+            has_wildcard = self._scheme_eval.scheme_eval(
+                f'(has-fnmatch-wild-card? "{name}")'
+            )
+        else:
+            has_wildcard = self._app_utilities.is_wildcard(name)
+        return self._scheme_eval.is_defined("has-fnmatch-wild-card?") and has_wildcard
```

Comment:
@hpohekar It looks like it shouldn't be too difficult to encapsulate this in the `AppUtilities` object.

Reply:
@seanpearsonuk What do we want to encapsulate in the `AppUtilities` object ?

Reply:
@hpohekar we want to hide the fact that there are different implementations depending on the Fluent version.